activesupport/lib/active_support/concern.rb

```diff
@@ -104,6 +104,13 @@ def initialize
       end
     end
 
+    class UnreachableIncludedBlock < StandardError #:nodoc:
+      def initialize
+        super "This 'included' block will never be run because the Concern was already included. " +
+          "Check your code for circular references from the Concern to the including class."
```

Comment:
"This 'included' block will never be run because the Concern was already included. This usually happens when the Concern references the Class that includes it, and this Concern is loaded before the Class. Check your code for circular references from the Concern to the including class and remove them to fix this." ?